Data/DataModel/PuzzleStatePerTeam.cs

```diff
@@ -68,5 +73,124 @@ public bool IsSolved
         /// Notes input by the team for this puzzle
         /// </summary>
         public string Notes { get; set; }
+
+        #region Consistency Helpers
```

Comment:
This is all starting to be a lot of logic in the model. Should we consider somewhere else to put this kind of code?

Reply:
Yep, will move to EventOperations as discussed before.

Reply:
Nope, will remove it except for the subset that the Status page needs, which will move there.